source/getting_started/how-to.rst

```diff
@@ -12,6 +12,8 @@ How to enable live boot
 
 The live boot feature allows you to flash a live boot image to a USB disk, and then boot from this USB disk to Android directly without installing Android to the internal storage first.
 
+The live boot is only used for debug, and only can be built in eng and userdebug build.
```

Comment:
Suggest to rewrite "The live boot feature is used only for debug purposes, and it must be built with the `eng` or `userdebug` build flavors."

PS: Use this symbol "`" to highlight the build flavors in different text format.